schemas/2018-01-01/Microsoft.EventGrid.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-01-01/Microsoft.EventGrid.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.EventGrid",
+  "description": "Microsoft EventGrid Resource Types",
+  "resourceDefinitions": {
+    "topics": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.EventGrid/topics"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2018-01-01"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource"
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Tags of the resource"
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/TopicProperties"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Properties of the topic"
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "apiVersion",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.EventGrid/topics"
+    }
+  },
+  "definitions": {
+    "TopicProperties": {
```

Comment:
>TopicProperties [](start = 5, length = 15)

is this an empty property bag? This can be a problem because an empty property bag matches any object.

Reply:
yes its empty, it has read-only properties, in next API version few properties will be added here. its similar to namespace in ServiceBus.

Reply:
I see. The PR looks good. Please sync the latest and resolve conflicts.

Reply:
resolved conflicts